Services/DLNAService.m

```diff
@@ -1147,4 +1164,84 @@ - (ServiceSubscription *) subscribeMuteWithSuccess:(MuteSuccessBlock)success fai
     return subscription;
 }
 
+#pragma Playlist controls
+
+- (void) playNextWithSuccess:(SuccessBlock)success failure:(FailureBlock)failure
+{
+    NSString *nextXML = @"<?xml version=\"1.0\" encoding=\"utf-8\"?>"
+    "<s:Envelope s:encodingStyle=\"http://schemas.xmlsoap.org/soap/encoding/\" xmlns:s=\"http://schemas.xmlsoap.org/soap/envelope/\">"
+    "<s:Body>"
+    "<u:Next xmlns:u=\"urn:schemas-upnp-org:service:AVTransport:1\">"
+    "<InstanceID>0</InstanceID>"
+    "</u:Next>"
+    "</s:Body>"
+    "</s:Envelope>";
+    
+    NSDictionary *stopPayload = @{
+                                  kActionFieldName : @"\"urn:schemas-upnp-org:service:AVTransport:1#Next\"",
+                                  kDataFieldName : nextXML
+                                  };
+    
+    ServiceCommand *stopCommand = [[ServiceCommand alloc] initWithDelegate:self target:_avTransportControlURL payload:stopPayload];
+    stopCommand.callbackComplete = ^(NSDictionary *responseDic){
+        if (success)
+            success(nil);
+    };
+    stopCommand.callbackError = failure;
+    [stopCommand send];
+}
+
+- (void) playPreviousWithSuccess:(SuccessBlock)success failure:(FailureBlock)failure
+{
+    NSString *previousXML = @"<?xml version=\"1.0\" encoding=\"utf-8\"?>"
+    "<s:Envelope s:encodingStyle=\"http://schemas.xmlsoap.org/soap/encoding/\" xmlns:s=\"http://schemas.xmlsoap.org/soap/envelope/\">"
+    "<s:Body>"
+    "<u:Previous xmlns:u=\"urn:schemas-upnp-org:service:AVTransport:1\">"
+    "<InstanceID>0</InstanceID>"
+    "</u:Previous>"
+    "</s:Body>"
+    "</s:Envelope>";
+    
+    NSDictionary *stopPayload = @{
+                                  kActionFieldName : @"\"urn:schemas-upnp-org:service:AVTransport:1#Previous\"",
+                                  kDataFieldName : previousXML
+                                  };
+    
+    ServiceCommand *stopCommand = [[ServiceCommand alloc] initWithDelegate:self target:_avTransportControlURL payload:stopPayload];
+    stopCommand.callbackComplete = ^(NSDictionary *responseDic){
+        if (success)
+            success(nil);
+    };
+    stopCommand.callbackError = failure;
+    [stopCommand send];
+}
+
+- (void)jumptoTrack:(NSInteger)trackNumber success:(SuccessBlock)success failure:(FailureBlock)failure
+{
+    NSString *trackNumberInString = [NSString stringWithFormat:@"%ld",(long)trackNumber];
```

Comment:
The indexing should be zero-based and it must be commented in the method description to avoid confusion.